bundles/org.openhab.binding.meteoalerte/README.md

```diff
@@ -69,26 +70,26 @@ Thing meteoalerte:department:yvelines @ "MyCity" [department="YVELINES", refresh
+Group gMeteoAlert "Alertes Météo" <meteo_france> 
+String     MA_Dept78                  "Département 78 [%s]"       <text>               (gMeteoAlert)   {channel="meteoalerte:department:yvelines:comment"}
+Number     MA_etat_canicule           "Canicule [%s]"             <canicule>           (gMeteoAlert)   {channel="meteoalerte:department:yvelines:canicule"}
+Number     MA_etat_grand_froid        "Grand Froid [%s]"          <grand-froid>        (gMeteoAlert)   {channel="meteoalerte:department:yvelines:grand-froid"}
+Number     MA_etat_pluie_inondation   "Pluie-Inondation [%s]"     <pluie-inondation>   (gMeteoAlert)   {channel="meteoalerte:department:yvelines:pluie-inondation"}
+Number     MA_etat_neige              "Neige [%s]"                <neige>              (gMeteoAlert)   {channel="meteoalerte:department:yvelines:neige"}
+Number     MA_etat_vent               "Vent [%s]"                 <vent>               (gMeteoAlert)   {channel="meteoalerte:department:yvelines:vent"}
+Number     MA_etat_inondation         "Inondation [%s]"           <inondation>         (gMeteoAlert)   {channel="meteoalerte:department:yvelines:inondation"}
+Number     MA_etat_orage              "Orage [%s]"                <orage>              (gMeteoAlert)   {channel="meteoalerte:department:yvelines:orage"}
+Number     MA_etat_avalanche          "Avalanches [%s]"           <avalanches>         (gMeteoAlert)   {channel="meteoalerte:department:yvelines:avalanches"}
     
-    Image       MA_icon_canicule         "Canicule"              <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:canicule-icon"}
-    Image       MA_icon_grand_froid      "Grand Froid"           <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:grand-froid-icon"}
-    Image       MA_icon_pluie_inondation "Pluie-Inondation"      <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:pluie-inondation-icon"}
-    Image       MA_icon_neige            "Neige"                 <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:neige-icon"}
-    Image       MA_icon_vent             "Vent"                  <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:vent-icon"}
-    Image       MA_icon_inondation       "Inondation"            <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:inondation-icon"}
-    Image       MA_icon_orage            "Orage"                 <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:orage-icon"}
-    Image       MA_icon_avalanche        "Avalanche"             <aqi>       (gMeteoAlert)   {channel="meteoalerte:department:yvelines:avalanches-icon"}
+Image      MA_icon_canicule           "Canicule"                  <canicule>           (gMeteoAlert)   {channel="meteoalerte:department:yvelines:canicule-icon"}
```

Comment:
Not sure it is necessary to specify an icon for the Image channels.

Reply:
I did it. It is just an example. The end user is allowed to do wathever he wants. I would keep this.